### PR TITLE
Retrieve channel message

### PR DESCRIPTION
**Pull request by magicjohnson**
* add use cases to enqueue and process channel notifications
* add channel_message_retriever to execute use case
* add SQS/ElasticMq repo to handle notifications queue

### CODE REVIEW

File: intergov/use_cases/process_channel_notifications.py [hunk 1]
```diff
@@ -0,0 +1,117 @@
...
+
+    def process(self, job_payload):
+        channel = job_payload[self.CHANNEL_ID]
+        notification_payload = job_payload[self.NOTIFICATION_PAYLOAD]
+        url = self._get_channel_message_endpoint_url(channel, notification_payload['id'])
+        message = self._get_message(url)
+        self.enqueue_message_use_case.execute(message)
+
+    def _get_message(self, url):
+        response = requests.get(url)
```
Comment (koriaf) [inline intergov/use_cases/process_channel_notifications.py:103]: and what about JWT auth we have for the cloud setup? multichannel worker seems to be handling it on per-channel basis, may be it's worth init this HttpApiChannel class like it does to handle it.

**Review by koriaf — APPROVED** [review]
looks fine apart the auth
@roleyfoley please be aware of the new queue and some env variables

File: intergov/use_cases/process_channel_notifications.py (hunk 1 above)
Comment (magicjohnson) [reply intergov/use_cases/process_channel_notifications.py:103]: currently shared-db-channel doesn't support auth. Let's implement it as separate user story as soon as we have clear requirements

Comment (koriaf) [reply intergov/use_cases/process_channel_notifications.py:103]: https://github.com/trustbridge/trustbridge-hamlet/tree/master/devnet this setup uses transparent auth - underlying code is not aware of this layer

https://sharedchannel.services.devnet.trustbridge.io/

**Comment by koriaf** [comment]
is our intergov setup down after these changes or it's something else to blame?

**Pull request merged**